.travis.yml

```diff
@@ -1,28 +1,33 @@
 sudo: required
-language: python
+language: java
 jdk: openjdk8
-services:
-  - docker
-python:
-  - "3.6"
-cache: pip
-matrix:
-  fast_finish: true
+services: docker
 
-env:
-  - T=nt  FL="-a star" NXF_VER=0.27.6
-  - T=nt  FL="-a star"
-  - T=nts FL="-a star -s"
-  - T=nt  FL="-a hisat2" NXF_VER=0.27.6
-  - T=nt  FL="-a hisat2"
-  - T=nts FL="-a hisat2 -s"
+before_install: docker pull nfcore/rnaseq:latest
 
 install:
-  - "./tests/install.sh $T"
-  - "cd ${TRAVIS_BUILD_DIR}/tests/"
+  # Install Nextflow
+  - mkdir /tmp/nextflow
+  - cd /tmp/nextflow
+  - wget -qO- get.nextflow.io | bash
+  - sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
+  # Install nf-core/tools
+  - git clone https://github.com/nf-core/tools.git /tmp/nf-core-tools
+  - cd /tmp/nf-core-tools
+  - pip install --user -e .
+  # Make test directories
+  - mkdir ${TRAVIS_BUILD_DIR}/tests
+  - cd ${TRAVIS_BUILD_DIR}/tests
+
+
+env:
+  - NXF_VER=0.30.1
+  - ''
 
 script:
+  # Lint the pipeline code
   - "nf-core lint ${TRAVIS_BUILD_DIR}"
-  - "./run_test.sh ${FL} -b"
-  - "./run_test.sh ${FL}"
-  - "find results/" # Print results files
+  # Run, build reference genome with STAR
+  - nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker
```

Comment:
You're only testing Docker?
Why not testing Conda and Singularity as well?

Reply:
Short answer is that I think it's unlikely that the pipeline will fail with one and not the others, except for annoying failures that resolve themselves when restarted. Of course there is a chance that we could break one container, but seeing as the builds are all based on conda then I think it's unlikely that we'll be messing around with them much. Also, we get email alerts when the builds fail, or should do..

Reply:
I agree, but since we're mainly using singularity it would make more sense to test with singularity as well at least.

Reply:
Sure, feel free to add a PR ;)